_posts/2020-05-30-f-droid-repo.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "Team NewPipe publishes own F-Droid repository"
+short: "NewPipe F-Droid repository"
+date: 2020-05-30 23:30:00 +0100
+categories: [announcement, f-droid, pinned]
+author: TheAssassin
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Team NewPipe proudly announces a self-hosted F-Droid repository, providing the official NewPipe app as well as -- in the future -- NewPipe Legacy. The F-Droid repository is a convenient way for users to update more quickly, which is especially useful whenever one of the supported services break.
+
+<!-- more -->
+
+## How to add the repository to F-Droid
+
+Instead of telling you the boring story behind the repository, let's first talk about how you can use it from F-Droid on your device(s).
+
+The easiest way to add the repository is to scan the QR code below or [click this link](fdroidrepo://archive.newpipe.net/fdroid/repo/?fingerprint=E2402C78F9B97C6C89E97DB914A2751FDA1D02FE2039CC0897A462BDB57E7501) if you're viewing this on your device already. F-Droid should pop up on both, prompting you to add the repository. For the paranoid (*which is not a bad idea, especially in this context!*), you can verify the fingerprint F-Droid shows you with the one in the "copy-pastable" URL below.
+
+<center>
+    <img src="/img/fdroid-repo-qrcode.svg" alt="NewPipe F-Droid repository QR code" style="width: 250px">
+</center>
+
+Alternatively, in F-Droid, open the *Settings*, and choose *Repositories*. Then press the *plus* button, and paste the link if needed.
+
+Once you added the repository, update your package lists. If you had NewPipe installed from the official F-Droid repository before, you will have to uninstall this one before F-Droid will display the packages from our repository.
```

Comment:
```suggestion
Once you added the repository, update your package lists. If you had NewPipe installed from the official F-Droid repository before, you will have to uninstall this one before F-Droid will display the packages from our repository. Please make sure to [backup and restore the ap data]({{ site.base_url }}/FAQ/tutorials/import-export-data/#export-subscriptions) to not lose your subscriptions, history and settings.

```

Reply:
not to lose your subscriptions* (tell me if I'm wrong)

Reply:
in order to avoid loosing your XXX